code/modules/tables/rack.dm

```diff
@@ -37,7 +37,7 @@
 /obj/structure/table/rack/retail_shelf
 	name = "retail shelves"
 	desc = "A large, assorted shelf with many platforms for setting things upon."
-	icon_state = "shelf"
+	icon_state = "shelf_alt"
```

Comment:
Shouldn't this be on a subtype of the shelves, so both types of shelf icon are usable?